src/pydvl/influence/frameworks/torch_differentiable.py

```diff
@@ -33,23 +32,26 @@ def flatten_gradient(grad):
     return torch.cat([el.reshape(-1) for el in grad])
 
 
-class TorchTwiceDifferentiable(TwiceDifferentiable):
+class TorchTwiceDifferentiable(TwiceDifferentiable[torch.Tensor, nn.Module]):
     """
     Calculates second-derivative matrix vector products (Mvp) of a pytorch torch.nn.Module
     """
 
     def __init__(
         self,
-        model: "nn.Module",
-        loss: Callable[["torch.Tensor", "torch.Tensor"], "torch.Tensor"],
+        model: nn.Module,
+        loss: Callable[[torch.Tensor, torch.Tensor], torch.Tensor],
     ):
         """
-        :param model: A torch.nn.Module representing a (differentiable) function f(x).
-        :param loss: Loss function L(f(x), y) maps a prediction and a target to a single value.
+        :param model: A torch.nn.Module representing a (differentiable) function.
```

Comment:
As mentioned before, the type is redundant
```suggestion
        :param model: A (differentiable) function.
```